Whisky/fr.lproj/Localizable.strings

```diff
@@ -52,3 +51,4 @@
 "rename.rename" = "Renommer";
 
 "check.updates" = "Recherche des Mises à Jour...";
+"kill.bottles" = "Kill All Bottles...";
```

Comment:
```suggestion
"kill.bottles" = "Fermer toutes les applications...";
```

Reply:
Suggestions should be made to the Crowdin [page](https://crowdin.com/project/whisky)